src/WebJobs.Script/Description/FunctionLogger.cs

```diff
@@ -1,109 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using Microsoft.Azure.WebJobs.Host;
-using Microsoft.Azure.WebJobs.Logging;
-using Microsoft.Extensions.Logging;
-
-namespace Microsoft.Azure.WebJobs.Script.Description
-{
-    // Each function can get its own log stream.
-    // Static per-function logging information.
-    public class FunctionLogger : IDisposable
-    {
-        private bool _disposed = false;
-
-        public FunctionLogger(ScriptHost host, string functionName, string logDirName = null)
-        {
-            // Function file logging is only done conditionally
-            TraceWriter traceWriter = host.FunctionTraceWriterFactory.Create(functionName, logDirName);
```

Comment:
All this complexity to handle logging to file only when enabled, only on the primary host, etc. is handled somewhere else now?

Reply:
Yeah, inside the FileLogger. It looks at the state passed in (similar to Properties in the TraceWriter) and if these flags are there, it behaves appropriately. I have some FileLoggerTests that verify.